### PR TITLE
386, 389 Talks URLs

### PR DESCRIPTION
**Pull request by dpoirier**
Fixes: #389, #386

### CODE REVIEW

File: symposion/schedule/tests/factories.py [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+
+class SlotKindFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = SlotKind
+
+    schedule = factory.SubFactory(ScheduleFactory)
+    label = factory.fuzzy.FuzzyText()
+
+
+class SlotFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = Slot
```
Comment (rebecca-caktus) [inline symposion/schedule/tests/factories.py]: On the most recent version of `factory_boy`, this is deprecated syntax and should be replaced:

```
class SlotFactory(factory.DjangoModelFactory):

    class Meta:
        model = Slot

    ....
```

Comment (dpoirier) [inline symposion/schedule/tests/factories.py]: Do you think we should update all our factories?  Maybe that should be another pull request.

**Comment by dpoirier** [comment]
Updated & ready to review again.

File: symposion/schedule/admin.py [hunk 2]
```diff
@@ -61,13 +65,25 @@ class PresentationAdmin(admin.ModelAdmin):
...
         'section__name',
+        'video_url',
+        'assets_url',
+        'slides_url',
     )
 
+    def get_queryset(self, request):
+        qs = super(PresentationAdmin, self).get_queryset(request)
+        return qs.select_related('speaker', 'speaker__user',
+                                 )
```
Comment (rebecca-caktus) [inline symposion/schedule/admin.py]: I think that the trailing parenthesis would be better read on the previous line.

File: pycon/registration/tests/test_views.py [hunk 3]
```diff
@@ -42,11 +43,9 @@ def setUp(self):
         self.user.user_permissions.add(self.permission)
         self.login_user(self.user)
 
+    @override_settings(ACCOUNT_CREATE_ON_SAVE=False)
     def create_user(self, **kwargs):
-        user = factories.UserFactory.build(**kwargs)
-        user._disable_account_creation = True
-        user.save()
-        return user
+        return factories.UserFactory(**kwargs)
```
Comment (rebecca-caktus) [inline pycon/registration/tests/test_views.py]: Nice!

**Comment by rebecca-caktus** [comment]
:sparkles:

File: symposion/schedule/admin.py (hunk 2 above)
Comment (dpoirier) [inline symposion/schedule/admin.py]: Agreed, not sure how that happened.
